cfme/fixtures/candu.py

```diff
@@ -1,28 +1,60 @@
 """
 Fixtures for Capacity and Utilization
 """
+import math
+from collections import namedtuple
+from datetime import date
+from datetime import datetime
+from datetime import timezone
+
 import fauxfactory
 import pytest
+from wrapanapi import VmState
 
+from cfme.base.credential import Credential
+from cfme.intelligence.chargeback.assignments import ComputeAssign
+from cfme.intelligence.chargeback.assignments import StorageAssign
 from cfme.utils import conf
+from cfme.utils.log import logger
 from cfme.utils.ssh import SSHClient
+from cfme.utils.wait import wait_for
+
+
+RATE_DIVISORS = {
+    'hourly': 1,
+    'daily': 24,
+    'weekly': 24 * 7,
+    'monthly': 24 * 30,
+    'yearly': 24 * 365
+}
+
+CostParam = namedtuple('CostParam', ['cost_key', 'metric_description', 'usage_key', 'rate_type'])
+RESOURCE_COST_PARAMS = [
+    CostParam('cpu_used_cost', 'Used CPU', 'cpu_used', 'Compute'),
+    CostParam('memory_used_cost', 'Used Memory', 'memory_used', 'Compute'),
+    CostParam('network_used_cost', 'Used Network I/O', 'network_used', 'Compute'),
+    CostParam('disk_used_cost', 'Used Disk I/O', 'disk_used', 'Compute'),
+    CostParam('storage_used_cost', 'Used Disk Storage', 'storage_used', 'Storage'),
+    CostParam('storage_alloc_cost', 'Allocated Disk Storage', 'storage_alloc', 'Storage'),
+    CostParam('memory_alloc_cost', 'Allocated Memory', 'memory_alloc', 'Compute'),
+    CostParam('cpu_alloc_cost', 'Allocated CPU Count', 'cpu_alloc', 'Compute')
+]
 
 
 @pytest.fixture(scope="module")
 def enable_candu(appliance):
+    # C&U data collection consumes a lot of memory and CPU.
+    # Disable server roles that are not needed for chargeback/
+    # metering report tests.
```

Comment:
Could this be included in the comment ?

# Disable server roles that are not needed for chargeback/
# metering report/ Cap and U tests.